src/main/resources/GoogleApps/RetrieveFromGoogleApps.xml

```diff
@@ -40,7 +40,7 @@
 #set($gagroovy = $xwiki.parseGroovyFromPage("GoogleApps.Groovy"))
 #set($ok = $gagroovy.init($xwiki, $context, $doc))
 #if($request.url)
- #set($ok = $gagroovy.retrieveFileFromGoogle($request.page, $request.name, $request.id, $request.url))
+ #set($ok = $gagroovy.retrieveFileFromGoogle($request.page, $request.name, $request.id, $request.url, $request.mt))
```

Comment:
I'm not fond of short (cryptic) parameter names. I would have used ``mediaType``.